### DISCUSSION TITLE
Any plans to migrate from Rust-tools to rustaceanvim?

### DISCUSSION THREAD

**Discussion by JasterV** [LazyVim/LazyVim#2575] [Q&A] [answered]
Hello there! 

I've just noticed that the [`rust-tools`](https://github.com/simrat39/rust-tools.nvim) project has been archived and won't be maintained anymore. Instead they highly encourage users to switch to https://github.com/mrcjkb/rustaceanvim.

As far as I see on the website the Rust extras feature still relies on `rust-tools`, are there any plans to move to the `rustaceanvim`?

**Comment by dpetka2001** [comment, marked as answer]
There is #2198 waiting to be merged.

**Reply by JasterV** [reply]
Thank you!
